docs/source/development/firewall.rst

```diff
@@ -0,0 +1,419 @@
+        ``dmz (VIDEOSERVER) -> inet (Attacker)`` | TCP Port 443 | (POLICY: `dmz -> inet ACCEPT`) - Sliver connection from ``VIDEOSERVER`` to Attacker.
+*   **SSH:**
+        ``inet (Attacker) -> dmz (VIDEOSERVER)`` | TCP Port 2222 | (DNAT) - Attacker uses the stolen SSH key (`privesc_key_videoserver`) to log into ``VIDEOSERVER`` via the DNAT rule.
+
+.. list-table:: Scenario 2 Firewall Connections
+   :widths: 25 20 20 15 20
+   :header-rows: 1
+
+   * - Phase / Action
+     - Source
+     - Destination
+     - Protocol / Port
+     - Firewall Rule/Policy
+   * - DNS Setup
+     - ``inet (Attacker)``
+     - ``inetdns``
+     - N/A
+     - Config Task
+   * - Initial Access
+     - ``inet (Attacker)``
+     - ``dmz (VIDEOSERVER)``
+     - TCP 80
+     - DNAT
+   * - DNS Lookup (Victim)
+     - ``dmz (VIDEOSERVER)``
+     - ``fw``
+     - TCP/UDP 53
+     - RULE: ``DNS/ACCEPT``
+   * - DNS Lookup (Firewall)
+     - ``fw``
+     - ``inetdns / inet``
+     - TCP/UDP 53
+     - POLICY: ``fw -> all ACCEPT``
+   * - Payload Download
+     - ``dmz (VIDEOSERVER)``
+     - ``inet (Attacker)``
+     - TCP 80
+     - POLICY: ``dmz -> inet ACCEPT``
+   * - Command & Control
+     - ``dmz (VIDEOSERVER)``
+     - ``inet (Attacker)``
+     - TCP `443`
+     - POLICY: ``dmz -> inet ACCEPT``
+   * - Post-Exploitation SSH
+     - ``inet (Attacker)``
+     - ``dmz (VIDEOSERVER)``
+     - TCP 2222
+     - DNAT
+
+Scenario 3: Lateral Movement
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This scenario involves brute-forcing credentials and moving laterally within the network.
+
+*   **Initial Access (Credential Attack):**
+        ``inet (Attacker) -> dmz (VIDEOSERVER)`` | TCP Port 80 | (DNAT) - Attacks against ZoneMinder login.
+
+        ``inet (Attacker) -> dmz (REPOSERVER)`` | TCP Port 10022 | (DNAT) - SSH brute-force attempt against ``REPOSERVER``.
+
+        ``inet (Attacker) -> dmz (REPOSERVER)`` | TCP Port 3389 | (DNAT) - RDP brute-force attempt against ``REPOSERVER``.
+
+        ``inet (Attacker) -> dmz (REPOSERVER)`` | TCP Port 5901 | (DNAT) - VNC brute-force attempt against ``REPOSERVER``.
+*   **Command and Control:**
+        ``dmz (VIDEOSERVER/REPOSERVER) -> inet (Attacker)`` | TCP Port 4444 | (POLICY: `dmz -> inet ACCEPT`) - Reverse shell from compromised DMZ hosts.
+*   **Lateral Movement (From compromised DMZ host, e.g., REPOSERVER):**
+        ``dmz (REPOSERVER) -> lan (KAFKA)`` | TCP Port 9092 | (RULE) - Accessing Kafka service.
+
+        ``dmz (REPOSERVER) -> fw`` | TCP Port 22 | (RULE: `SSH/ACCEPT`) - SSH connection to the firewall.
+
+        ``dmz (REPOSERVER) -> fw`` | TCP/UDP Port 53 | (RULE: `DNS/ACCEPT`) - DNS lookups via the firewall.
+
+        ``dmz (REPOSERVER) -> dmz (VIDEOSERVER)`` | Various Ports | (Intra-zone, allowed) - Communication within the DMZ.
```

Comment:
I don't think that the reposerver communicates with the videoserver

Reply:
I miss the connections reposerver(dmz) -> linuxshare(lan) and linuxshare(lan) -> reposerver(dmz)